airbyte-cdk/java/airbyte-cdk/dependencies/src/testFixtures/kotlin/io/airbyte/workers/general/DbtTransformationRunner.kt

```diff
@@ -119,25 +119,24 @@ class DbtTransformationRunner(
                     emptyMap(),
                     *dbtArguments.toTypedArray<String>()
                 )
+            this.process = process
```

Comment:
Why not assign to `this.process` directly? Evidently there's a reason but for my own information I'd like to understand.